build.gradle

```diff
@@ -1763,6 +1763,7 @@ project(':tools') {
     implementation libs.jacksonJDK8Datatypes
     implementation libs.slf4jApi
     implementation libs.log4j
+    implementation libs.joptSimple
```

Comment:
I believe we should stop adding this to pull requests 😄. It should have been merged in https://github.com/apache/kafka/pull/13136/files if not before.

Reply:
Yes this PR needs to be rebased on trunk to remove the extra changes like this and `ToolsTestUtils`/`TransactionCommandTest`.